lindris.com.domain-authentication.json

```diff
@@ -0,0 +1,56 @@
+{
+  "providerId": "lindris.com",
+  "providerName": "Lindris",
+  "serviceId": "domain-authentication",
+  "serviceName": "Lindris Email Domain Authentication",
+  "version": 1,
+  "logoUrl": "https://app.lindris.com/images/logo.png",
+  "description": "Configures DNS records for domain verification and email authentication to send emails through Lindris",
+  "variableDescription": "subdomain is the email subdomain prefix (e.g., sl, el1, mail); mail_server is the target Lindris mail server (e.g., m1.lindris.net, m2.lindris.net, eu.lindris.net); dkim_selector is the DKIM key selector (e.g., key1 for shared keys, or unique identifier for dedicated keys); verification_id and verification_hash are unique credentials for domain ownership verification.",
+  "syncPubKeyDomain": "lindris.com",
+  "syncRedirectDomain": "app.lindris.com",
+  "warnPhishing": false,
+  "records": [
+    {
+      "groupId": "email",
+      "type": "CNAME",
+      "host": "%subdomain%",
+      "pointsTo": "%mail_server%",
+      "ttl": 3600,
+      "essential": "Always"
+    },
+    {
+      "groupId": "dkim",
+      "type": "CNAME",
+      "host": "%dkim_selector%._domainkey",
```

Comment:
I would like to see one thing clarified.
In the example this record and all other records are under %subdomain%.
is it changed now that dkim is actually under APEX? As I see DMARC is also on a subdomain so I would expect DKIM to be the same.
If all records are under same host, why would you use %subdomain% as special variable instead of %host%?
What people tend to miss is that even if same records would be generated for both approaches, in the latter case it is possible to provision template 2 times on 2 different subdomains. If just a variable is used this could result in removing the first one with some providers. Remedy can be multiInstance setting.